larq/models.py

```diff
@@ -36,6 +38,35 @@
     *mac_containing_layers,
 )
 
+
+def is_mac_containing_layer(layer: tf.keras.layers.Layer):
+    # Check if the layer is either one of the above, or a SavedModel version of it.
+    if isinstance(layer, mac_containing_layers):
+        return True
+
+    if version.parse(tf.__version__) >= version.parse("2.0"):
+        from tensorflow.python.keras.saving.saved_model.load import RevivedLayer
+
+        names = {cls.__name__ for cls in mac_containing_layers}
+        return isinstance(layer, RevivedLayer) and layer.__class__.__name__ in names
+
+    return False
+
+
+def is_op_count_supported_layer(layer: tf.keras.layers.Layer):
+    # Check if the layer is either one of the above, or a SavedModel version of it.
+    if isinstance(layer, op_count_supported_layer_types):
+        return True
+
+    if version.parse(tf.__version__) >= version.parse("2.0"):
+        from tensorflow.python.keras.saving.saved_model.load import RevivedLayer
+
+        names = {cls.__name__ for cls in op_count_supported_layer_types}
+        return isinstance(layer, RevivedLayer) and layer.__class__.__name__ in names
+
+    return False
```

Comment:
These functions are almost the same. If I am right you can just add one variable as argument and replace both functions by one, e.g. something like:
```suggestion
def is_of_layer_type(layer: tf.keras.layers.Layer, layer_list: List) -> bool:
    # Check if the layer is either one of the above, or a SavedModel version of it.
    if isinstance(layer, layer_list):
        return True

    if version.parse(tf.__version__) >= version.parse("2.0"):
        from tensorflow.python.keras.saving.saved_model.load import RevivedLayer

        names = {cls.__name__ for cls in layer_list}
        return isinstance(layer, RevivedLayer) and layer.__class__.__name__ in names

    return False
```